tiller_python_service/git_actions.py

```diff
@@ -10,6 +10,36 @@
 import re
 import openai
 
+def my_sorting_function(arr):
+    n = len(arr)
```

Comment:
This code is very inefficient, could we replace its implementation with something with a better runtime?